src/Ombi.Core/Rule/Rules/Search/AvailabilityRuleHelper.cs

```diff
@@ -79,66 +79,6 @@ public static void CheckForUnairedEpisodes(SearchTvShowViewModel search)
                 log.LogError(e, "Exception thrown when attempting to check if something is available");
             }
 
-            if (epExists != null)
```

Comment:
why was all of this removed?

Reply:
git diff is a bit stupid, the last bits of this method are still there after the part I removed (l.142/82).
If you were asking about the whole section being removed: the Plex/Jellyfin/Emby methods were merged into a single one which takes IMediaServerEpisode as an input.